pom.xml

```diff
@@ -367,11 +367,10 @@ under the License.
 				<version>3.2.2</version>
 			</dependency>
 
-			<!-- common-beanutils-bean-collections is used by flink-shaded-hadoop2 -->
 			<dependency>
 				<groupId>commons-beanutils</groupId>
-				<artifactId>commons-beanutils-bean-collections</artifactId>
-				<version>1.8.3</version>
+				<artifactId>commons-beanutils</artifactId>
+				<version>1.9.3</version>
```

Comment:
The 1.8.x releases of BeanUtils have distributed three jars:
- commons-beanutils.jar - contains everything
- commons-beanutils-core.jar - excludes Bean Collections classes
- commons-beanutils-bean-collections.jar - only Bean Collections classes

Version 1.9.0 reverts this split for reasons outlined at [BEANUTILS-379](http://issues.apache.org/jira/browse/BEANUTILS-379). There is now only one jar for the BeanUtils library.